index.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Portfolio page</title>
+	<meta name="description" content="Holly's portfolio page">
+	<meta name="author" content="Holly Bedford">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css" type="text/css">
+	<link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css" type="text/css">
+</head>
+
+<body>
+	<header>
+		<a class="btn" href="#contacts">Contact</a>
+		<h3>Portfolio page</h3>
+		<a class="btn" href="#items">Portfolio stuff</a>
+		
+	</header>
+	
+	<div class="banner"></div>
+	
+	<main>
+		<p class="quote">"Spends too much time with the cat"</p>
+		<a class="btn" href="#about-me">About me</a>
+	</main>
+	
+	<div class="banner">Portfolio Stuff</div>
+	
+	<section class="items" id="items">
```

Comment:
You probably don't need both a class and id with the same name 😄 . Can you style with just the class? Additionally, as ids are unique, it's generally best practice to also make the names of ids unique too